tools/dds/dds-server/rs-dds-server.cpp

```diff
@@ -96,205 +97,175 @@ void start_streaming( std::shared_ptr< tools::lrs_device_controller > lrs_device
     } );
 }
 
-void add_init_device_header_msg( rs2::device dev, std::shared_ptr<realdds::dds_device_server> &server )
-{
-    using namespace realdds::topics;
-   
-    device::notification::device_header_msg device_header_msg;
+void add_init_device_header_msg( rs2::device dev, std::shared_ptr< dds_device_server > &server )
+{   
+    topics::device::notification::device_header_msg device_header_msg;
     auto &&sensors = dev.query_sensors();
-    device_header_msg.num_of_sensors = sensors.size();
+    std::map<std::pair<rs2_stream, int>, int> unique_streams;
```

Comment:
Can you add the comment with the map fields?

Reply:
Done